src/env/uat/terraform.tfvars

```diff
@@ -322,3 +324,13 @@ reporting_function_autoscale_default = 1
 reporting_batch_function_always_on    = true
 reporting_service_function_always_on  = true
 reporting_analysis_function_always_on = true
+
+# canone unico
+canoneunico_plan_kind     = "Linux"
```

Comment:
Since this value does not change among the environment you could set it as a default in varriable.tf file

Reply:
done 👍